static/js/zamboni/static_theme.js

```diff
@@ -134,13 +134,18 @@ $(document).ready(function() {
         var $color = $wizard.find('input.color-picker');
         $color.change(function() {
             var $this = $(this),
-                $svg_element = $('.' + $this[0].id);
+                color_class = '.' + $this[0].id,
```

Comment:
Need a `var`. Also, maybe call this `color_property_selector` or something similar instead instead: since it includes the `.` it's not technically a class name, and well, not technically just a color either :)

Reply:
it's in the var - see the `,` on the line above.  I'll rename.

Reply:
https://github.com/mozilla/addons-server/pull/9859/commits/c5609381e7cfe6c168adedaabdbb793947a0bfe6

Reply:
Oh yeah, damn multi-line declarations